src/tools/dart-create.md

```diff
@@ -0,0 +1,113 @@
+---
+title: dart create
+description: Command-line tool for creating Dart projects.
+toc: false
+---
+
+The `dart create` command allows creating Dart projects
+from varying available templates through a command line
+or many IDE project creation workflows.
+
+{% include tools/dart-tool-note.md %}
+
+The command creates a directory with the specified name, 
+creates necessary files, configures static analysis, 
+and retrieves any necessary dependencies.
+
+```terminal
+$ dart create [ARGUMENTS] <DIRECTORY>
+```
+
+{{site.alert.note}}
+  The command will attempt to run [`dart pub get`](/tools/pub/cmd/pub-get) 
+  after the project is created.
+
+  To avoid this you can specify the `--no-pub` flag:
+
+  ```terminal
+  $ dart create --no-pub <DIRECTORY>
+  ```
+{{site.alert.end}}
```

Comment:
This doesn't seem crucial enough to go above the fold. What do you think of deleting this note in favor of the `--no-pub` mention above?